salt/modules/glusterfs.py

```diff
@@ -217,8 +217,8 @@ def peer(name):
     return _gluster(cmd)
 
 
-def create_volume(name, bricks, stripe=False, replica=False, device_vg=False,
-           transport='tcp', start=False, force=False):
+def create_volume(name, bricks, stripe=False, replica=False, arbiter=False,
```

Comment:
Can you move the new kwarg to the end of the list? We don't want to create an API change with this new addition.

Reply:
Done.